main.tf

```diff
@@ -100,12 +100,118 @@ module "aws_deploy-main-us-east-2" {
   additional_storage_size = 30
 
   aeternity = {
-    package = "${var.package}"
+    package = var.package
   }
 
   providers = {
     aws = "aws.us-east-2"
   }
 
-  depends_on = ["${module.aws_deploy-main-us-west-2.static_node_ips}"]
+  dependency = module.aws_deploy-main-us-west-2.static_node_ips
+}
+
+# Monitoring nodes
+
+module "aws_deploy-main_mon-ap-southeast-1" {
+  source            = "github.com/aeternity/terraform-aws-aenode-deploy?ref=v2.0.0"
+  env               = "main_mon"
+  bootstrap_version = var.bootstrap_version
+  vault_role        = "ae-node"
+  vault_addr        = var.vault_addr
+
+  static_nodes = 0
+  spot_nodes   = 1
+
+  spot_price    = "0.03"
+  instance_type = "t3.medium"
+  ami_name      = "aeternity-ubuntu-16.04-v1559735157"
+
+  additional_storage      = true
+  additional_storage_size = 30
+
+  aeternity = {
+    package = var.package
+  }
+
+  providers = {
+    aws = "aws.ap-southeast-1"
+  }
+}
+
+module "aws_deploy-main_mon-us-west-2" {
+  source            = "github.com/aeternity/terraform-aws-aenode-deploy?ref=v2.0.0"
+  env               = "main_mon"
+  bootstrap_version = var.bootstrap_version
+  vault_role        = "ae-node"
+  vault_addr        = var.vault_addr
+
+  static_nodes = 0
+  spot_nodes   = 1
+
+  spot_price    = "0.03"
+  instance_type = "t3.medium"
+  ami_name      = "aeternity-ubuntu-16.04-v1559735157"
+
+  additional_storage      = true
+  additional_storage_size = 30
+
+  aeternity = {
+    package = var.package
+  }
+
+  providers = {
+    aws = "aws.us-west-2"
+  }
+}
+
+module "aws_deploy-main_mon-us-east-2" {
+  source            = "github.com/aeternity/terraform-aws-aenode-deploy?ref=v2.0.0"
+  env               = "main_mon"
+  bootstrap_version = var.bootstrap_version
+  vault_role        = "ae-node"
+  vault_addr        = var.vault_addr
+
+  static_nodes = 0
+  spot_nodes   = 1
+
+  spot_price    = "0.03"
+  instance_type = "t3.medium"
+  ami_name      = "aeternity-ubuntu-16.04-v1559735157"
+
+  additional_storage      = true
+  additional_storage_size = 30
+
+  aeternity = {
+    package = var.package
+  }
+
+  providers = {
+    aws = "aws.us-east-2"
+  }
+}
+
+module "aws_deploy-main_mon-eu-north-1" {
+  source            = "github.com/aeternity/terraform-aws-aenode-deploy?ref=v2.0.0"
+  env               = "main_mon"
+  bootstrap_version = var.bootstrap_version
+  vault_role        = "ae-node"
+  vault_addr        = var.vault_addr
+
+  static_nodes = 0
+  spot_nodes   = 1
+
+  spot_price    = "0.03"
+  instance_type = "t3.medium"
+  ami_name      = "aeternity-ubuntu-16.04-v1559735157"
+
+  additional_storage      = true
+  additional_storage_size = 30
```

Comment:
We recommend 40
https://github.com/aeternity/aeternity/blob/9e251f87532934df535f315c265ac9887fd6fc6a/docs/installation.md#deploy-node